frontend/app/Global.scala

```diff
@@ -21,17 +16,4 @@ object Global extends WithFilters(CheckCacheHeadersFilter, CSRFFilter(), Gzipper
     TouchpointBackend.All.foreach(_.start())
     GuardianContentService.start()
   }
-
-  override def onHandlerNotFound(request: RequestHeader): Future[Result] = {
```

Comment:
Whoops, shouldn't have thrown this away, although there's a nicer way of doing this in Play 2.4 : https://github.com/guardian/membership-frontend/pull/587